driver/flambda_backend_args.ml

```diff
@@ -729,6 +766,9 @@ module Extra_params = struct
       end;
       true
     in
+    let clear' option =
```

Comment:
(Could share code with `set'`.)

Reply:
Don't think this is worth it -- we'll have to rewrite all this for the `-X` proposal anyway...